app/js/constants/droits.js

```diff
@@ -1,302 +1,322 @@
-        ],
-        'imgSrc': 'logo_mairie_paris.png',
-        'link': 'http://www.paris.fr/aides',
-        'isBaseRessourcesYearMoins2': false,
-        'roundToNearest10': false
-    },
-    'paris_energie_famille': {
-        'id': 'paris_energie_famille',
-        'isMontantAnnuel': true,
-        'label': 'Paris Energie Famille',
-        'shortLabel': 'PEF',
-        'description': 'Paris énergie familles est une aide annuelle permettant aux familles d’alléger leurs dépenses de consommation d’électricité et de gaz.',
-        'imgSrc': 'logo_mairie_paris.png',
-        'link': 'http://www.paris.fr/aides',
-        'isBaseRessourcesYearMoins2': false,
-        'roundToNearest10': false
-    },
-    'paris_complement_sante': {
-        'id': 'paris_complement_sante',
-        'isMontantAnnuel': true,
-        'label': 'Complémentaire Santé Paris',
-        'shortLabel': 'CSP',
-        'description': 'Le complément santé paris est une allocation qui permet de participer aux frais d’adhésion à une couverture complémentaire santé (mutuelle ou autre organisme analogue)',
-        'imgSrc': 'logo_mairie_paris.png',
-        'link': 'http://www.paris.fr/aides',
-        'isBaseRessourcesYearMoins2': false,
-        'roundToNearest10': false
-    },
-    'adpa': {
-        'id': 'adpa',
-        'isMontantAnnuel': false,
-        'unit': '%',
-        'legend': 'des frais de dépendance',
-        'label': 'Allocation Départementale Personnalisée d’Autonomie',
-        'shortLabel': 'ADPA',
-        'description': 'Allocation versée aux personnes dépendantes âgées de plus de 60 ans. Elle a pour objectif de financer des services d’aide à la personne favorisant leur autonomie dans les gestes quotidiens.',
-        'conditions': [],
-        'imgSrc': 'logo_cd93.png',
-        'link': 'https://www.seine-saint-denis.fr/ADPA.html',
-        'isBaseRessourcesYearMoins2': false
-    },
-    'aah': {
-        'id': 'aah',
-        'isMontantAnnuel': false,
-        'label': 'Allocation aux adultes handicapés',
-        'shortLabel': 'AAH',
-        'imgSrc': 'logo_caf.png',
-        'description': 'L’AAH est une aide financière qui permet d’assurer un revenu minimum aux adultes handicapés.',
-        'isBaseRessourcesYearMoins2': true,
-        'uncomputability': {
-            'intervention_CDAPH_necessaire': {
-                'reason': 'le montant de l’AAH dépend du taux d’incapacité déterminé par la <abbr title="Commission des droits et de l’autonomie des personnes handicapées">CDAPH</abbr> après le dépôt de votre demande',
-                'solution': 'Contactez <a href="http://informations.handicap.fr/carte-france-mdph.php">la <abbr title="Maison départementale des personnes handicapées">MDPH</abbr> la plus proche</a> pour faire établir ce taux. Attention, votre éligibilité à l’AAH peut impacter vos droits au RSA et à la CMU-C.'
-            }
+    ],
+    'partenairesLocaux': [
+        {
+            'id': 'paris',
```

Comment:
I wanted to be able to choose the order and put Paris first.
For prestations, before they were objects with and id key, which was redundant.
Is it better to have an object ?